packages/base-styles/_z-index.scss

```diff
@@ -24,6 +24,7 @@ $z-layers: (
 	".block-editor-warning": 5,
 	".block-library-gallery-item__inline-menu": 20,
 	".block-editor-url-input__suggestions": 30,
+	".edit-post-layout__content": 35,
```

Comment:
If the goal is to fix the snackbar position, I would consider changing the z-index of `components-snackbar-list` rather than the content container itself. It might have adverse effects otherwise. 

Unfortunately this particular classname also changed in a recent refactor in #18044, so it no longer exists in `master`.